plasma_framework/contracts/src/exits/payment/PaymentInFlightExitModelLib.sol

```diff
@@ -0,0 +1,88 @@
+pragma solidity ^0.5.0;
+
+import "../../utils/Bits.sol";
+import { PaymentExitDataModel as ExitModel } from "./PaymentExitDataModel.sol";
+
+library PaymentInFlightExitModelLib {
+    using Bits for uint256;
+
+    uint8 constant public MAX_INPUT_NUM = 4;
+    uint8 constant public MAX_OUTPUT_NUM = 4;
+
+    function setInputPiggybacked(ExitModel.InFlightExit storage ife, uint16 index)
+        internal
+    {
+        ife.exitMap = ife.exitMap.setBit(uint8(index));
+    }
+
+    function setOutputPiggybacked(ExitModel.InFlightExit storage ife, uint16 index)
+        internal
+    {
+        uint8 indexInExitMap = uint8(index + MAX_INPUT_NUM);
```

Comment:
Hmm we assume that `2 * MAX_INPUT_INDEX` is 8-bit integer, but declare index as twice as big :question: Seems incorrect or I'm missing the point.... :nerd_face: